modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RustServerCodegen.java

```diff
@@ -1056,6 +1056,26 @@ public boolean isDataTypeFile(final String dataType) {
         return dataType != null && dataType.equals(typeMapping.get("File").toString());
     }
 
+    /**
+     * Add operation to group
+     *
+     * @param tag          name of the tag
+     * @param resourcePath path of the resource
+     * @param operation    OAS Operation object
+     * @param co           Codegen Operation object
+     * @param operations   map of Codegen operations
+     */
+    @SuppressWarnings("static-method")
+    public void addOperationToGroup(String tag, String resourcePath, Operation operation, CodegenOperation
+            co, Map<String, List<CodegenOperation>> operations) {
+        // only generate operation for the first tag of the tags
+        if (tag != null && co.tags.size() > 1 && !tag.equals(co.tags.get(0).getName())) {
+            LOGGER.warn("generated skip operationId `" + co.operationId + "` with the repeating tag=" + tag);
```

Comment:
I don't think this is a warning - this is a perfectly fine situation to exist.

Also, I think we should say `additional`, not `repeating`.

Reply:
LOGGER.info("generated skip additional tag `" + tag + "` with operationId=" + co.operationId);